Modix/ClientApp/package-lock.json

```diff
@@ -10,9 +10,9 @@
       "integrity": "sha512-zN69TnSr0viRSU6cEDIcuPcP67QcpQ6uHACg58FiN9PDrU6SLyGW3MR4tiISbYxy1kDWAVPwD+XwQTWE5cigAA==",
       "dev": true,
       "requires": {
-        "cssnano": "^4.0.0",
```

Comment:
Is there some sort of npm setting we should be aware of that causes this disconnect? The bot builds fine on my machine without changing the `package-lock.json` file, and I'm guessing that the opposite is true for you (i.e. that it gets replaced with the file you have here).

Reply:
Initial research indicates that `npm ci` is preferable to `npm install` for consistency - the former will only restore packages from the lockfile, while the latter might modify them.

Reply:
Thaaaaat makes sense. When I was looking into it, to try and determine if we even NEED that file as part of the repo, the description I got was basically that the `package-lock.json` file is kinda like a materialized `package.json` file, which is deterministic. That is, a build from a `package.json` file does not always produce the same output, because it depends on the state of the remote sources, while a build from `package-lock.json` will always produce the same output.

It definitely sounds like `package-lock.json` should not be getting rebuilt on every build. It's the equivalent of having NuGet auto-update package versions to the latest, during every build.